@babel/runtime dependency while consuming Zustand with Typescript

When importing Zustand in a typescript (node CLI) project, I'm getting this error:

```
Error: Cannot find module '@babel/runtime/regenerator'
Require stack:
- /Users/me/projects/project/node_modules/zustand/index.cjs.js
```

There seem to be an `index.js` and an `index.cjs.js`, with Typescript importing the latter. The index.cjs.js requires @babel/runtime. My Node version is 12.x and my tsconfig looks like this:
```
{
    "compilerOptions": {
        "lib": ["es2019", "es2020.promise", "es2020.bigint", "es2020.string"],
        "module": "commonjs",
        "target": "es2019",
        "strict": true,
        "esModuleInterop": true,
        "skipLibCheck": true,
        "forceConsistentCasingInFileNames": true,
        "jsx": "react"
    }
}
```

I rather not change my tsconfig or Node version, unless there is a compelling reason. Is it correct that for my case I need to install `@babel/runtime` as a dependency? I would expect not to, since `tsc` should probably take care of everything?

Thanks!

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -13,29 +13,21 @@ function external(id) {
   return !id.startsWith('.') && !id.startsWith(root)
 }
 
+function getBabelOptions(targets) {
+  return {
+    ...createBabelConfig({ env: (env) => env === 'build' }, targets),
+    extensions,
+    comments: false,
+    babelHelpers: 'bundled',
+  }
+}
+
 function getEsbuild(target) {
   return esbuild({
     minify: false,
     target,
     tsconfig: path.resolve('./tsconfig.json'),
   })
-}
-
-function getBabelOptions(targets) {
-  const config = createBabelConfig({ env: (env) => env === 'build' }, targets)
-  if (targets.ie) {
-    config.plugins = [
-      ...config.plugins,
-      '@babel/plugin-transform-regenerator',
-      ['@babel/plugin-transform-runtime', { helpers: true, regenerator: true }],
-    ]
-    config.babelHelpers = 'runtime'
-  }
-  return {
-    ...config,
-    comments: false,
-    extensions,
-  }
 }
 
 function createDeclarationConfig(input, output) {

